.changeset/nine-eels-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen-react': patch
```

Comment:
Are we doing everything as patch for now? Where can I read more about releasing @frehner (I need a refresh on it).

Reply:
https://github.com/Shopify/hydrogen-ui/blob/2022-10/README.md#versioning

Reply:
Perfect, thank you